### PR TITLE
made changes for rebranding with case changes

### PR DESCRIPTION
**Pull request by badri-pathak**
## Pull request checklist of the issue [Changes related to rebranding of Spectrum Scale](https://github.ibm.com/ibmspectrumscale/csi-tracker/issues/284)

```release-notes
There is a requirement of "Spectrum Scale" rebranding. The existing name (Spectrum Scale) is going to be Storage Scale.
```

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature Enhancement
- [ ] Test Automation 
- [ ] Code Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Community Operator listing 
- [x] Other (please describe):  This is the rebranding change according to new delivery decision. 

## What is the current behavior?

- The current published name is "Spectrum Scale"

## What is the new behavior?

- In the upcoming release, this would streamline with CNSA regarding name. The new name will be "Storage Scale". Resource names will remain same because it is required for upgrade. Logs and description will be changed to new name.

## How risky is this change?
- [ ] Small, isolated change
- [x] Medium, requires regression testing
- [ ] Large, requires functional and regression testing

### CODE REVIEW

**Comment by badri-pathak** [comment]
REgression passed: 
image

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Contributing
 
-Work on the `ibm-spectrum-scale-csi` project should always be performed in a forked copy of the repo, incorporated into the main project using a pull request against the Development (dev) branch. 
+Work on the `ibm-storage-scale-csi` project should always be performed in a forked copy of the repo, incorporated into the main project using a pull request against the Development (dev) branch.
```
Comment (deeghuge) [inline CONTRIBUTING.md:3]: we are not changing git repo name to keep it spectrum only

File: cloudpak/stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/case.yaml [hunk 2]
```diff
@@ -2,9 +2,9 @@ name: ibm-spectrum-scale-csi-operator
 specVersion: 1.0.0
 version: 2.10.0
 appVersion: 2.10.0
-description: "Represents a deployment of the IBM CSI Spectrum Scale driver."
-displayName: "IBM CSI Spectrum Scale Driver CASE" 
-displayDescription: "Represents a deployment of the IBM CSI Spectrum Scale driver."
+description: "Represents a deployment of the IBM CSI Storage Scale driver."
```
Comment (deeghuge) [inline cloudpak/stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/case.yaml:5]: @Jainbrt please check this part if this works for case

Comment (badri-pathak) [reply cloudpak/stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/case.yaml:5]: changed in the last commit.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -26,8 +26,8 @@ Environmental output
 - What openshift/kubernetes version are you running, and the architecture? 
 - `kubectl get pods -o wide -n < csi driver namespace> `
 - `kubectl get nodes -o wide`
-- CNSA/Spectrum Scale version 
-- Remote Spectrum Scale version 
+- CNSA/Storage Scale version
```
Comment (whowutwut) [inline .github/ISSUE_TEMPLATE/bug_report.md:29]: Even though internally we use "CNSA", for our external facing documentation, we try and avoid using this and it's not an official acronym.  Suggest making this two bullets.  Note the capitalization, container native is not capitalized, intentionally. 

- IBM Storage Scale container native version
- IBM Storage Scale version

File: cloudpak/stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/case.yaml (hunk 2 above)
Comment (Jainbrt) [reply cloudpak/stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/case.yaml:5]: @deeghuge as discussed we can raise a new PR for case bundle.

File: driver/csiplugin/connectors/rest_v2.go [hunk 4]
```diff
@@ -235,12 +235,12 @@ func (s *SpectrumRestV2) GetScaleVersion(ctx context.Context) (string, error) {
...
 	err := s.doHTTP(ctx, getVersionURL, "GET", &getVersionResponse, nil)
 	if err != nil {
-		klog.Errorf("[%s] unable to get Spectrum Scale version: [%v]", utils.GetLoggerId(ctx), err)
+		klog.Errorf("[%s] unable to get Storage Scale version: [%v]", utils.GetLoggerId(ctx), err)
 		return "", err
 	}
 
 	if len(getVersionResponse.Info.ServerVersion) == 0 {
-		return "", fmt.Errorf("unable to get spectrum scale version")
+		return "", fmt.Errorf("unable to get storage scale version")
```
Comment (Jainbrt) [inline driver/csiplugin/connectors/rest_v2.go:243]: we should keep consistency while keeping storage or Storage ( capital S) ?

File: driver/csiplugin/controllerserver.go [hunk 5]
```diff
@@ -782,7 +782,7 @@ func (cs *ScaleControllerServer) CreateVolume(ctx context.Context, req *csi.Crea
 		}
 	}
 
-	klog.Infof("[%s] volume:[%v] -  spectrum scale volume create params : %v\n", loggerId, scaleVol.VolName, scaleVol)
+	klog.Infof("[%s] volume:[%v] -  storage scale volume create params : %v\n", loggerId, scaleVol.VolName, scaleVol)
```
Comment (Jainbrt) [inline driver/csiplugin/controllerserver.go:785]: we should keep consistency while keeping storage or Storage ( capital S) ?

File: driver/csiplugin/nodeserver.go [hunk 6]
```diff
@@ -90,7 +90,7 @@ func (ns *ScaleNodeServer) NodePublishVolume(ctx context.Context, req *csi.NodeP
 	}
 	volScalePath := volumeIDMembers.Path
 
-	klog.V(4).Infof("[%s] Target SpectrumScale Path : %v\n", loggerId, volScalePath)
+	klog.V(4).Infof("[%s] Target StorageScale Path : %v\n", loggerId, volScalePath)
```
Comment (Jainbrt) [inline driver/csiplugin/nodeserver.go:93]: we should keep consistency as Storage Scale rather StorageScale.

File: driver/csiplugin/settings/scale_config.go [hunk 7]
```diff
@@ -104,13 +105,13 @@ func LoadScaleConfigSettings(ctx context.Context) ScaleSettingsConfigMap {
 
 	file, e := ioutil.ReadFile(ConfigMapFile) // TODO
 	if e != nil {
-		klog.Errorf("[%s] spectrum scale configuration not found: %v", utils.GetLoggerId(ctx), e)
+		klog.Errorf("[%s] storage scale configuration not found: %v", utils.GetLoggerId(ctx), e)
```
Comment (Jainbrt) [inline driver/csiplugin/settings/scale_config.go:108]: we should keep consistency while keeping storage or Storage ( capital S) ?

File: driver/csiplugin/connectors/rest_v2.go (hunk 4 above)
Comment (badri-pathak) [reply driver/csiplugin/connectors/rest_v2.go:243]: Looks like internal logs only, we can update. But would be fine only, will print in the middle of the statement.

**Comment by Jainbrt** [comment]
@badri-pathak is this PR still valid?

**Comment by badri-pathak** [comment]
> @badri-pathak is this PR still valid?

@Jainbrt As we have discussed earlier, we need to raise a new PR after making changes related to case. 
Also olm-catalog directory has been removed, so there won't be any files under the same. 
For OLM/case, we will have to create a separate issue to remove OLM based distribution. @amdabhad

**Comment by amdabhad** [comment]
@badri-pathak , there are 2 issues created on zenhub:
https://github.ibm.com/IBMSpectrumScale/csi-tracker/issues/369
https://github.ibm.com/IBMSpectrumScale/csi-tracker/issues/393

Please discuss with Abhishek if anything else is required for rebranding related to cloudpak and close this PR.

**Pull request closed without merging**